src/components/learner-credit-management/BudgetAssignmentsTable.jsx

```diff
@@ -1,23 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { DataTable } from '@edx/paragon';
-
+import {
+  Button,
+  DataTable,
+} from '@edx/paragon';
 import TableTextFilter from './TableTextFilter';
 import CustomDataTableEmptyState from './CustomDataTableEmptyState';
 import AssignmentDetailsTableCell from './AssignmentDetailsTableCell';
 import AssignmentStatusTableCell from './AssignmentStatusTableCell';
+import AssignmentRowActionTableCell from './AssignmentRowActionTableCell';
 import { DEFAULT_PAGE, PAGE_SIZE, formatPrice } from './data';
 import AssignmentRecentActionTableCell from './AssignmentRecentActionTableCell';
 
 const FilterStatus = (rest) => <DataTable.FilterStatus showFilteredFields={false} {...rest} />;
 
+const CancelAction = ({ selectedFlatRows, ...rest }) => (
+  // eslint-disable-next-line no-console
+  <Button variant="danger" onClick={() => console.log('Cancel', selectedFlatRows, rest)}>
+    {`Cancel (${selectedFlatRows.length})`}
+  </Button>
+);
+
+const RemindAction = ({ selectedFlatRows, ...rest }) => (
+  // eslint-disable-next-line no-console
+  <Button onClick={() => console.log('Remind', selectedFlatRows, rest)}>
+    {`Remind (${selectedFlatRows.length})`}
```

Comment:
[clarification] Should we be assuming all selected rows can reminded? For example, if an admin selects a row with the "Status" column depicting "Notifying learner", "Failed: Bad email", etc., I'm figuring we wouldn't want to allow reminders on those rows. 

Related, I see these annotations in the Figma:

<img width="270" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/28cdc248-2ac2-4bda-ac7f-53bad1da5612">
<br />
<br />

<img width="357" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/58364cd3-5d24-4797-91eb-8b8668a4678e">

Is this something that intends to be addressed as part of this PR or are there plans around the conditional logic for these bulk actions as follow-up work?

Reply:
you're totally right in that we should not allow the reminding of assignments that aren't pending, however current UX is to hide the button, not disable

Reply:
I've added the behavior according to the AC on the ticket

Reply:
The first screenshot you have is from out an out of date mock

Reply:
Just a heads up, this behavior as implemented may need to be revisited after hearing from Mike Brown from UX about the intended bulk action behavior.

![image](https://github.com/openedx/frontend-app-admin-portal/assets/2828721/cea7d90f-4645-446f-bcb1-0afb9fa0ded1)

The Figma mocks above allude that the "Assigned" table bulk actions should largely have parity with subscription management's bulk actions. However, the behavior as spec'd/implemented (hiding the bulk action when non-actionable rows are selected) is **not** parity with subscription bulk actions.

There are possible usability concerns with hiding the bulk action (e.g., possible lack of discoverability of the bulk action, forcing admins to manually unselect one or more rows for the bulk action to re-appear). These were mitigated in subscriptions bulk actions by keeping the bulk action visible but exclude non-actionable rows. If there are no actionable rows, the bulk action becomes disabled.

Mike/UX is revisiting this interaction now. We may need to update this behavior in code once a final decision is made.